guides/customization/vnc.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using VNC with Coder
+description: Learn how to use a VNC with Coder
+---
+
+This guide will show you how to set up a Virtual Network Computing (VNC) system
+in Coder. Coder does not have a specific set of VNC providers it supports. Coder
+will render the VNC, as long as it is installed on the image used to create the
+environment.
+
+## Step 1: Create a Dockerfile image with a VNC provider installed
```

Comment:
Should we specify `Dockerfile` here, or just a generic image? If the latter, this will affect the commands described in Step 2.